diff -crB include_backup/CollectorMap.h include/CollectorMap.h
*** include_backup/CollectorMap.h	2012-11-16 09:50:50.290679659 -0800
--- include/CollectorMap.h	2012-11-16 09:52:44.006678325 -0800
***************
*** 696,702 ****
  
          CollectorIter Itr2 = _list.upper_bound(key);
          while(Itr1 != Itr2) {
!           destroy(&Itr1->second);
            ++Itr1;
          }
          return (_list.erase(key));
--- 696,702 ----
  
          CollectorIter Itr2 = _list.upper_bound(key);
          while(Itr1 != Itr2) {
!           RemovalPolicy<T>::destroy(&Itr1->second);
            ++Itr1;
          }
          return (_list.erase(key));
***************
*** 753,759 ****
        void selfDestruct() {
          CollectorIter itr;
          for(itr = _list.begin() ; itr != _list.end() ; itr++) {
!           destroy(&itr->second);
          }
          _list.clear();
        }
--- 753,759 ----
        void selfDestruct() {
          CollectorIter itr;
          for(itr = _list.begin() ; itr != _list.end() ; itr++) {
!           RemovalPolicy<T>::destroy(&itr->second);
          }
          _list.clear();
        }
Only in include: CollectorMap.h~
