diff -crB ./src/base/objs/SparseBlockMatrix/SparseBlockMatrix.cpp ../isis/src/Core/SparseBlockMatrix/SparseBlockMatrix.cpp
*** .src/base/objs/SparseBlockMatrix/SparseBlockMatrix.cpp      2012-06-14 15:31:09.000000000 -0700
--- ../isis/src/Core/SparseBlockMatrix/SparseBlockMatrix.cpp    2012-06-14 15:54:15.000000000 -0700
***************
*** 5,5 ****
--- 5,6 ----
  #include <boost/numeric/ublas/io.hpp>
+ #include <boost/numeric/ublas/matrix.hpp>
